docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -66,9 +66,10 @@ $ kubectl apply -f ./resources/default-sa-rbac-role.yaml
 $ ./scripts/install-tiller.sh
 ```
 
-5. Configure the Kyma installation using the local configuration file:
+5. Configure the Kyma installation using the local configuration file from the latest release:
 ```
-$ kubectl apply -f https://github.com/kyma-project/kyma/releases/download/0.4.1/kyma-config-local.yaml
+$ LATEST=$(curl https://github.com/kyma-project/kyma/releases/latest -I|grep Location:| rev | cut -d'/' -f1 | rev|tr -d '\r')
```

Comment:
this is wrong assumption, this is not how releases work 
latest is not latest with higher number, latest is a release that was created the latest
so we can have 0.5 release but then patch to 0.4 like 0.4.6 and this one will be marked as latest
example `curl https://github.com/derberg/test/releases/latest -I|grep Location:| rev | cut -d'/' -f1 | rev|tr -d '\r'` returns 1.5 and not 2.0, and I guess you would like `2.0`

so if you want to have some easy approach here, you need to base it on tags and take highest number. Consult with @michal-hudy as he worked in similar topic when we were enabling versioning for kyma-project.io docs website